data.sql

```diff
@@ -1,32 +1,15 @@
-INSERT INTO animals (ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg ) 
-VALUES (1 ,'Agumon', '2020-02-03', 0, 'true', '10.23');
+/* Populate database with sample data. */
 
-INSERT INTO animals ( ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg )
-VALUES ( 2, 'Gabumon', '2018-11-15', 2, 'true', '8' );
 
-INSERT INTO animals ( ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg )
-VALUES ( 3, 'Pikachu', '2021-01-7', 1, 'true', '15.04' );
+INSERT INTO  animals(name,date_of_birth,escape_attempts, neutered , weight_kg) VALUES('Agumon', 'Feb 03 2020',0, true, 10.23);
+INSERT INTO  animals(name,date_of_birth,escape_attempts, neutered , weight_kg) VALUES('Gabumon', 'Nov 15 2018',2, true, 8);
+INSERT INTO  animals(name,date_of_birth,escape_attempts, neutered , weight_kg) VALUES('Pikachu', 'Jan 7 2021',1, false, 15.04);
+INSERT INTO  animals(name,date_of_birth,escape_attempts, neutered , weight_kg) VALUES('Devimon', 'May 12 2017',5, true, 11);
 
-INSERT INTO animals ( ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg )
-VALUES ( 4, 'Devimon', '2017-05-12', 5, 'true', '1' );
-
-INSERT INTO animals ( ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg,Species )
-VALUES ( 5, 'Charmander', '2020-02-08', 0, 'false', '-11', 'Unspecified' );
-
-INSERT INTO animals ( ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg,Species )
-VALUES ( 6, 'Plantmon', '2021-11-15', 2, 'true', '-5.7', 'Unspecified' );
-
-INSERT INTO animals ( ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg,Species )
-VALUES ( 7, 'Squirtle', '1993-04-02', 3, 'true', '-12.13', 'Unspecified' );
-
-INSERT INTO animals ( ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg,Species )
-VALUES ( 7, 'Angemon', '2005-06-12', 1, 'true', '-45', '' );
-
-INSERT INTO animals ( ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg,Species )
-VALUES ( 8, 'Boarmon', '2005-01-07', 7, 'true', '20.4', '' );
-
-INSERT INTO animals ( ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg,Species )
-VALUES ( 10, 'Blossom', '1998-10-13', 3, 'true', '17', '' );
-
-INSERT INTO animals ( ID, Name, Date_of_birth, Escape_attempts, Neutered, Weight_kg,Species )
-VALUES ( 11, 'Ditto', '2022-05-14', 4, '', '22', '' );
\ No newline at end of file
+INSERT INTO  animals(name,date_of_birth,escape_attempts, neutered , weight_kg) VALUES('Charmander', 'Feb 08 2020',0, false, 11),
+('Plantmon', 'Nov 15 2021',2, true, 5.7),
+('Squirtle', 'Apr 02 1993',3, false, 12.13),
+('Angemon', 'Jun 12 2005',1, true, 45), 
+('Boarmon', 'Jun 07 2005',7, true, 20.4),
+('Blossom', 'Oct 13 1998',3, true, 17),
+('Ditto', 'May 14 2022',4, true, 22);
```

Comment:
- [ ] Kindly make sure all the animals have the correct data like the one provided in the **[project's requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md)** like the `Weight_kg ` column for **Charmander, Plantmon, Squirtle, Angemon** is a negative number but here you use positive ones. 

- [ ] Once you have updated the above changes, make sure to run the below transitions in order for your data to remain as expected as you did before. From your previous screenshot the number of updated animals was **0** because no one was less than **0**.
```PostgreSQL
- Inside a transaction:
  - Create a savepoint for the transaction.
  - Update all animals' weight to be their weight multiplied by -1.
  - Rollback to the savepoint
  - Update all animals' weights that are negative to be their weight multiplied by -1.
  - Commit transaction
 -Take a screenshot of the results of your actions.**
```